.coveragerc

```diff
@@ -1,3 +1,4 @@
 [run]
 branch = True
 include = eventtracking*
+omit = */*performance*, */*integration*
```

Comment:
There was no CLI alternative for excluding these test files from the coverage report.

Reply:
@danialmalik I think [-k expression](https://docs.pytest.org/en/stable/example/markers.html#using-k-expr-to-select-tests-based-on-their-name) should work